hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -436,51 +436,44 @@ set_false_path -from [get_ports IOC7] -to [get_ports IOR*]
 
 # first, set maximum delay from any FF to a 2 FF synchronizer to 0.9 x the shortest clock period in the system.
 set MAX_DELAY [expr $MAIN_TCK_PERIOD * 0.9]
-set SYNC_PATH u_sync_1/gen_*u_impl*/gen_flops[0]*.u_size_only_reg/Q
-set_max_delay $MAX_DELAY -from [get_pins -hierarchical *] -to [get_pins -hierarchical *$SYNC_PATH]
+set_max_delay $MAX_DELAY -to [get_pins -hierarchical -filter "full_name =~ *u_sync_1/gen_*u_impl*/gen_flops?0?*?u_size_only_reg/QN"]
 
 # for prim_sync_reqack_data instances
 # these are all through paths with data just being routed throgh data_i and data_o with no FF in between.
-set_max_delay $MAX_DELAY -from [get_pins -hierarchical *] -to [get_pins -hierarchical *]   \
-                         -through [get_pins -hierarchical *u_prim_sync_reqack_data/data_i] \
+set_max_delay $MAX_DELAY -through [get_pins -hierarchical *u_prim_sync_reqack_data/data_i] \
                          -through [get_pins -hierarchical *u_prim_sync_reqack_data/data_o]
-set_max_delay $MAX_DELAY -from [get_pins -hierarchical *] -to [get_pins -hierarchical *] \
-                         -through [get_pins -hierarchical *u_otp_key_req_sync/data_i]    \
-                         -through [get_pins -hierarchical *u_otp_key_req_sync/data_o]
-set_max_delay $MAX_DELAY -from [get_pins -hierarchical *] -to [get_pins -hierarchical *] \
-                         -through [get_pins -hierarchical *u_otp_key_req_sync/data_i]    \
+set_max_delay $MAX_DELAY -through [get_pins -hierarchical *u_otp_key_req_sync/data_i] \
                          -through [get_pins -hierarchical *u_otp_key_req_sync/data_o]
 
 # this one has a data reg
-set_max_delay $MAX_DELAY -from [get_pins -hierarchical *] -to [get_pins -hierarchical *u_prim_sync_reqack_data_in/*] \
+set_max_delay $MAX_DELAY -to [get_pins -hierarchical -filter "full_name =~ */u_prim_sync_reqack_data_in/*data_q*/QN"] \
```

Comment:
@arnonsha just to double check: is the QN pin the flop input pin? 
I updated a couple of constraints for async paths through a hierarchical pin (like `data_i`) into a flop.

Reply:
I will be surprised if /QN is the input - it should be the inverted output and /D should be the input (but I've not actually seen the lib).

Reply:
Q/QN are outputs
D is input

Reply:
Thanks - just wanted to confirm.